pkg/metrics/builder/builder_test.go

```diff
@@ -175,6 +175,9 @@ func TestMetricBuilderCollector(t *testing.T) {
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
 			metricBuilder, err := NewMetricBuilder(tc.opts, tc.value, tc.labelKeyValues)
+			if err != nil {
+				assert.Failf(t, err.Error(), "error constructing new metric builder")
+			}
```

Comment:
For better code readability and failure message, use `assert.NoError`.
```suggestion
			if !assert.NoError(t, err, "error constructing new metric builder") {
				return
			}
```

Reply:
@staebler Done, as per your suggestion.

Reply:
Make the same change [here](https://github.com/openshift/installer/pull/4028/files#diff-f44d33885aeb734e0cf56242b3b37752cd2f6b6127106e63922bbdd72df654d0R226).